### ISSUE TITLE
Chrome does not attempt to navigate to the provided URL when launched from the extension

### ISSUE THREAD

**Issue by dschuessler** [microsoft/vscode-chrome-debug#964]
- VS Code Version: 1.40.2
- Log file (set `"trace": true` in launch config): https://gist.github.com/dschuessler/a092d6bd60fd9bbf53450572da73327c

Steps to reproduce:

1. Create a launch configuration for Chrome in the `launch.json` like so but with a matching `url` and `webRoot`

```
    {
      "type": "chrome",
      "request": "launch",
      "name": "Debug Frontend",
      "url": "http://localhost:4200",
      "webRoot": "${workspaceFolder}/client/web"
    }
```

2. Launch Chrome with that configuration.

Expected behavior: Chrome navigates to the url in the configuration.

Actual behavior: Chrome does not attempt to navigate to the url in the configuration.

![Chrome bug 2019-12-13 17_16_10](https://user-images.githubusercontent.com/5774545/70815100-2ab18680-1dcd-11ea-98a8-dac697a19f72.gif)

Let me know if I can provide any further information you need to debug this.

**Comment by roblourens** [comment]
It's failing to attach to the browser. There is probably another chrome instance open using the same debug port. I think that's what the error message is trying to say.

**Comment by dschuessler** [comment]
There was indeed some kind of a zombie Chrome process running and as I terminated it, it worked. Thanks for your help.

Maybe you'd like to consider rephrasing the error message. It reads:

> Connection with runtime process not possible, Timeout after 10000 ms – (Cause: No valid target was found, that matches "about:blank". Available sites: ["http://localhost:9876/?id=36853390"]).

...which I would not consider helpful in this case.

**Comment by roblourens** [comment]
Glad you figured it out! This can happen for many reasons that can't all be listed here, and I think the "available sites" list is a good hint that there is another browser with that tab open somewhere.
